cfn-resources/network-peering/cmd/resource/resource.go

```diff
@@ -368,15 +349,23 @@ func validateCreationProcess(client *mongodbatlas.Client, currentModel *Model) h
 	)
 }
 
-func getStatus(client *mongodbatlas.Client, projectID, peerID string) (statusName string, errorStatusName string, err error) {
-	peerResponse, resp, err := client.Peers.Get(context.Background(), projectID, peerID)
+func getStatus(client *util.MongoDBClient, projectID, peerID string) (statusName string, errorStatusName string, err error) {
+	peerResponse, _, err := client.AtlasV2.NetworkPeeringApi.GetPeeringConnection(context.Background(), projectID, peerID).Execute()
```

Comment:
I see this function was in this way before, but returning 3 elements normally means that the logic could be structured better.

Reply:
Updated. Unfortunately, the commit is still not available here 😢